Functions.Templates/Templates/DurableFunctionsActivity-CSharp/build.config/template.json

```diff
@@ -0,0 +1,58 @@
+﻿{
+  "author": "Microsoft",
+  "classifications": [
+    "Azure Function"
+  ],
+  "name": "DurableFunctionsActivityCSharp",
+  "identity": "Azure.Function.CSharp.DurableFunctionsActivityTrigger",
+  "groupIdentity": "Azure.Function.DurableFunctionsActivityTrigger",
+  "shortName": "EventHub",
```

Comment:
>EventHub [](start = 16, length = 8)

I assume the use of "EventHub" here is a copy/paste omission. Should this be "Durable Activity" or maybe "Activity"?